server/db/postgres/seed.js

```diff
@@ -3,7 +3,9 @@ const fs = require('fs');
 const path = require('path');
 const log = require('fancy-log');
 const queriesArray = require('./dataGenerator.js').queriesArray;
```

Comment:
This seems like it's no longer exported from datagenerator, is this still working as intended?